### PR TITLE
DTSPO-5732: Adding panorama records to private dns

### PR DESCRIPTION
**Pull request by madjava**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSPO-5732

### Change description ###
Adding Panorama vms ips to dns zones

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: environments/sandbox/sandbox-platform-hmcts-net.yml [hunk 1]
```diff
@@ -125,6 +125,14 @@ A:
...
     - 10.50.11.10
     ttl: 300
+  - name: panorama-uks-0
+    record:
+    - 10.48.0.68
+    ttl: 300
+  - name: panorama-ukw-0
+    record:
+    - 10.100.198.164
+    ttl: 300
```
Comment (timja) [inline environments/sandbox/sandbox-platform-hmcts-net.yml:135]: couldn't you just add via terraform? or by enabling auto registration on the zone?

Comment (madjava) [reply environments/sandbox/sandbox-platform-hmcts-net.yml:135]: hey @timja, the terraform reads the `yml` file, looking at the repo. Is there another way to add this?

Comment (timja) [reply environments/sandbox/sandbox-platform-hmcts-net.yml:135]: add them in your terraform code or enable auto registration on the vnet you are deploying this into

**Comment by madjava** [comment]
Managing private DNS configuration from projects terraform configuration

**Pull request closed without merging**